adhocracy4/comments/models.py

```diff
@@ -60,18 +60,20 @@ def save(self, *args, **kwargs):
     def get_absolute_url(self):
         if hasattr(self.content_object, 'get_absolute_url'):
             return self.content_object.get_absolute_url()
-        elif hasattr(self.project, 'get_absolute_url'):
-            return self.project.get_absolute_url()
         else:
-            return None
+            return self.module.get_absolute_url()
 
     @property
     def notification_content(self):
         return self.comment
 
     @property
     def project(self):
```

Comment:
Mhm ... I am not really convinced. At least we might provide a default implementation in a useful place.